dashboard/test/ui/step_definitions/blockly.rb

```diff
@@ -1,20 +1,20 @@
 Given(/^block "([^"]*)" is at a location "([^"]*)"$/) do |block, identifier|
   @locations ||= {}
-  blockId = get_block_id(block)
-  @block = @browser.find_element(:css, "g[block-id='#{blockId}']")
-  x = @browser.execute_script("return $(\"[block-id='#{blockId}']\").position().left")
-  y = @browser.execute_script("return $(\"[block-id='#{blockId}']\").position().top")
+  block_id = get_block_id(block)
+  @block = @browser.find_element(:css, "g[block-id='#{block_id}']")
+  x = @browser.execute_script("return $(\"[block-id='#{block_id}']\").position().left")
+  y = @browser.execute_script("return $(\"[block-id='#{block_id}']\").position().top")
```

Comment:
This and changes like it in this file are just to conform to ruby naming conventions.